docs/source/resources/scylladbclusters/index.md

```diff
@@ -0,0 +1,8 @@
+# ScyllaDBClusters
+
+:::{toctree}
+:maxdepth: 1
+
+basics
+../common/exposing
```

Comment:
this is listed under both ScyllaCluster and ScyllaDBCluster

![image](https://github.com/user-attachments/assets/8b84ce01-9447-43e6-94d5-4b7a76d033ef)

is this desired?

Reply:
yes, this document is now shared

Reply:
With `ScyllaCluster` and `ScyllaDBCluster` being proper nouns for different concepts now, "Exposing **ScyllaDB cluster**" is an unfortunate name because it looks like a hidden association with `ScyllaDBCluster`.

That said, we can easily correct this post-release.
I'll create a follow up PR to address that.